index.html

```diff
@@ -100,24 +103,24 @@
         >
         <a href="#"><span>See all sponsors</span></a>
       </div>
-      <div class="mw-section-3-body">
-        <div class="mw-section-3-body-left">
-          <div class="mw-section-3-body-left-image">
+      <div class="section-3-body">
+        <div class="section-3-body-left">
+          <div class="section-3-body-left-image">
```

Comment:
hai sa refactorizam is noi clasele sa inceapa toate cu mw-s

Ex. 
```
section-1
mw-s1

mw-s2

mw-s3

mw-s4

mw-s5
...
```
 am vazut ca le ai amestectate